src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/DeleteAllButton.java

```diff
@@ -0,0 +1,44 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.common;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import mx4j.log.Logger;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'mx4j.log.Logger'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)